#Quantitative PCR Assay Development for Field Differentiation and Detection of Dispersal of the Hop Powdery Mildew Fungus#
##Cameron Ross1, Michele Wiseman1, Nanci Adair2, & David Gent1,2 #
###Oregon State University Department of Botany and Plant Pathology1, Forage Seed & Cereal Research Unit Corvallis, OR; USDA-ARS2#

This project attempts to create a quantitative PCR (qPCR) assay capable in field detection and quantification of Podosphaera macularis, the causal agent of Hop Powdery Mildew and use this assay to understand disease dispersal. To do this three things must occur: a novel region must be discovered in the diseases genes and compared to similar species, a qPCR assay will have to be designed from specific primers in this region, and the assay will have to be used against spore traps set up in regional transects and tested in order to understand dispersal patterns. The first step, finding a novel region will require sequence data of different candidate genes for multiple different Podosphaera spp. This means that the main dataset will be nucleotide sequences and comparisons between species. The primary data will consist of agarose gel images, and sequences either compiled from in house PCR reactions read on Geneious or extracted from NCBI. This allows for visual observations upon differences between these species. Next the assay design data will consist of “normalized fluorescence” graphs generated by the qPCR machine created from various quantities of DNA and spores such as 1, 10, 100, or 500 nanomolar and micromolar, enabling a relative quantification based on the fluorescent peak generated. Lastly using spore traps each week small 2cm stainless steel rods will be ran in this qPCR generating more graphs that will also contain additional locality information for each graph. Using this locality information, new normalized fluorescence graphs and previously created normalized fluorescence graphs quantification based on location can be inferred and disease dispersal patterns can be analyzed based upon temporal and spatial data acquired during collection of the traps. Nucleotide data will be collected through CGRB sequencing, NCBI database, and comparisons as well as primer design will be made in Geneious. All qPCR data will be generated through the thermocycler in the form of a normalized fluorescence graph. The locality, temporal and spatial data will be collected each week upon collection of the stainless-steel rods by recording time, date, yard location and trap number. Nucleotide data will consist of roughly 8 sequences acquired for each of the 25 Podosphaera spp. collected (approximately 200 nucleotide sequences). In the qPCR design roughly 35 to 50 replicates will occur and since there are 8 for both spores and DNA this could surmount to over 400 graphs generated as well as roughly 20 or more nanodrop graphs. Lastly depending on how many spore traps we deploy, most likely 16, compounded by 12 weeks means that at least another 200 or so normalized fluorescent graphs may be generated. Overall this project will have at least 800 pieces of unique data and will likey be in the single digits of GB.

